packages/webdriverio/src/commands/element/isDisplayed.js

```diff
@@ -41,5 +41,8 @@
  */
 
 export default function isDisplayed() {
-    return this.isElementDisplayed(this.elementId)
+    return this.parent.$(this.selector).then((elem) => {
```

Comment:
I think we can use async / await here.

Reply:
I agree

Reply:
Also I would propose to try/catch the isDisplayed command and return false if the command fails due to the error that the element does not exist

Reply:
Done.